<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberBox</class>
 <widget class="QWidget" name="MemberBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="add_button">
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="member_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="remove_button">
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
